nsis/src/nsis/format/NsisEntry.java

```diff
@@ -38,4 +38,22 @@ public DataType toDataType() throws DuplicateNameException, IOException {
 		return STRUCTURE;
 	}
 
+	/**
+	 * Get the size of the Entry structure
+	 * 
+	 * @return
+	 */
+	public static int getEntrySize() {
+		return STRUCTURE.getLength();
+	}
+
+	/**
+	 * Get the opcode of the entry
+	 * 
+	 * @return
+	 */
+	public int getOpCode() {
```

Comment:
I found this confusing, the function `getOpCode()` returning the value of a variable called `which`.

I see where in the NSIS code it came from.

I wonder if it makes sense to diverge from the names used in the source code?

I definitely think we need to some how explain that an `entry` is an `instruction` and the `which` is the opcode and the `offsets` are the argument offsets (at least I think this is correct).

Reply:
that's correct

I'm thinking of keeping the `which` in the STRUCTURE object, but naming the `which` parameter `opCode`. Also, I forgot to add a comment about the naming convention in the `static{}` bloc.

I'll make the changes and let me know what you think

Reply:
Looking good.